### PR TITLE
feat(license-api): Updates and revoke function for driving licenses

### PR DESCRIPTION
**Pull request by thorkellmani**
## What

* Implement the push/pull update and revoke

## Why

RLS needs to be able to update PkPass licenses on-demand

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/services/license-api/src/app/modules/license/clients/drivingLicense/drivingLicenseApiClient.service.ts [hunk 1]
```diff
@@ -7,26 +7,88 @@ import {
...
+
+  async pullUpdate(nationalId: string): Promise<Result<Pass | undefined>> {
+    let data
+    try {
+      data = await Promise.all([
+        this.drivingLicenseApi.getCurrentLicenseV4({ nationalId }),
+        this.drivingLicenseApi.getRemarksCodeTable(),
+      ])
+    } catch (e) {
+      return {
```
Comment (osk) [inline apps/services/license-api/src/app/modules/license/clients/drivingLicense/drivingLicenseApiClient.service.ts:40]: Log error

File: apps/services/license-api/src/app/modules/license/clients/drivingLicense/drivingLicenseMapper.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
+import {
```
Comment (osk) [inline apps/services/license-api/src/app/modules/license/clients/drivingLicense/drivingLicenseMapper.ts:1]: Shouldn't we use the same mappers for license-service and license-api?

**Review by osk — CHANGES_REQUESTED** [review]
More logging

File: apps/services/license-api/src/app/modules/license/clients/drivingLicense/drivingLicenseApiClient.service.ts (hunk 1 above)
Comment (thorkellmani) [reply apps/services/license-api/src/app/modules/license/clients/drivingLicense/drivingLicenseApiClient.service.ts:40]: ups

File: apps/services/license-api/src/app/modules/license/clients/drivingLicense/drivingLicenseMapper.ts (hunk 2 above)
Comment (thorkellmani) [reply apps/services/license-api/src/app/modules/license/clients/drivingLicense/drivingLicenseMapper.ts:1]: This'll be absorbed in #9718

**Review by osk — APPROVED** [review]

**Review by busla — APPROVED** [review]
LGTM

**Pull request merged**